modules/services/window-managers/i3-sway/i3.nix

```
@@ -143,51 +143,46 @@ let
     floatingCriteriaStr windowCommandsStr colorSetStr windowBorderString
     fontConfigStr keybindingDefaultWorkspace keybindingsRest workspaceOutputStr;
 
-  startupEntryStr = { command, always, notification, workspace, ... }: ''
-    ${if always then "exec_always" else "exec"} ${
-      if (notification && workspace == null) then "" else "--no-startup-id"
-    } ${
-      if (workspace == null) then
+  startupEntryStr = { command, always, notification, workspace, ... }:
+    concatStringsSep " " [
+      (if always then "exec_always" else "exec")
+      (if (notification && workspace == null) then "" else " --no-startup-id")
```

I don't think that space is necessary since inside of a `concatStringsSep`, right?
```suggestion
      (if (notification && workspace == null) then "" else "--no-startup-id")
```